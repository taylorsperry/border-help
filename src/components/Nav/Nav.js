import React from 'react'
import { NavLink } from 'react-router-dom'

export const Nav = ( {getRights, getScenarios} ) => {
  return (
    <div>
      <NavLink 
        onClick={getRights}
        exact to='/rights'
      >
        Your Rights
      </NavLink>
      <NavLink
        onClick={getScenarios}
        exact to='/scenarios'
      >
      What to Do
      </NavLink>
      <div>Help</div>
    </div>
  )
}

export default Nav