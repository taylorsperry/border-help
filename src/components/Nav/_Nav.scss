@import '../../variables';

.nav-container {
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  background-color: $blue;
  box-shadow: 0 4px 6px -5px rgba(0,0,0,0.20);
  margin-bottom: 50px;
}

.nav-link {
  color: $white;
  display: flex;
  align-items: center;
  justify-content: center;
  text-decoration: none;
  border: solid $navy 1px;
  width: 33.3%;
  height: 50px;
    &:hover {
      background-color: $blue-hover;
    }
}

.active {
  background-color: $red;
  &:hover {
    background-color: $red-hover
  }
}