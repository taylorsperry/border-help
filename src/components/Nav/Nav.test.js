import React from 'react'
import { Nav } from './Nav'
import { shallow } from 'enzyme'

describe('Nav', () => {
  let wrapper
  it('should match the snapshot', () => {
    wrapper = shallow(<Nav />)
    expect(wrapper).toMatchSnapshot()
  })
})