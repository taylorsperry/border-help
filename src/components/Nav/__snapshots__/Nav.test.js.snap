// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should match the snapshot 1`] = `
<div
  className="nav-container"
>
  <NavLink
    className="nav-link"
    exact={true}
    to="/rights"
  >
    Your Rights
  </NavLink>
  <NavLink
    className="nav-link"
    exact={true}
    to="/what-to-do"
  >
    What to Do
  </NavLink>
  <NavLink
    className="nav-link"
    exact={true}
    to="/help"
  >
    Help
  </NavLink>
  <NavLink
    className="nav-link"
    exact={true}
    to="/location"
  >
    Your Location
  </NavLink>
</div>
`;
